src/versions/v0.0.11.md

```
@@ -0,0 +1,12 @@
+# v0.0.11
+
+## Node
+### Helm
+Several performance modifications were made to improve deployment and a fix was applied to allow UI sign in when 
+ingress is disabled
+
+
+### UI
+#### Private S3 Bucket Data Stores
+Private buckets are now supported when creating a data store using an S3 bucket. Full documentation on how to do this 
+can be found [here](https://docs.privateaim.net/guide/admin/bucket-setup-for-data-store.html).
```

markdownlint-cli2 (0.22.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)